bluej/src/main/java/bluej/stride/framedjava/frames/TryFrame.java

```diff
@@ -140,9 +140,14 @@ else if (sectionIndex == 0)
             // "try" section
             text = " in the 'try' section,";
         }
+        else if(c == ((TryFrame)(c.getParent())).getTailCanvas())

```

Comment:
In this method, c.getParent() is always going to equal "this", so you can simplify to just `if (c == getTailCanvas())`